### ISSUE TITLE
siard skeleton file: does it contain the empty directory??

### ISSUE THREAD

**Issue by richardlehane** [exponential-decay/skeleton-container-test-suite-generator#8]
[fmt-1196-container-signature-id-31020.zip](https://github.com/exponential-decay/skeleton-container-test-suite-generator/files/4094379/fmt-1196-container-signature-id-31020.zip)

I can never get the SIARD skeleleton file to match. It is driving me mad! It looks OK ... i.e. when you decompress it "has" that *darned* directory. 

But I don't think it really does have that directory at all... 

...it just has a single empty file that has that directory as part of its filename.

BEHOLD!!! ...
![Screenshot from 2020-01-22 00-33-39](https://user-images.githubusercontent.com/157422/72852474-e4debc80-3cae-11ea-9906-c887946d7208.png)

**Comment by richardlehane** [comment]
didn't remember this last night, but this issue was raised previously by @marhop in the thread on the sf issue: https://github.com/richardlehane/siegfried/issues/123#issuecomment-426205023

**Comment by ross-spencer** [comment]
Right! I was wondering, but then on my end, I suspect I need to merge this [branch](https://github.com/exponential-decay/skeleton-container-test-suite-generator/tree/dev/fix-zip-generation). Some housekeeping to do on a few things this next two weeks,will try to take a look!

**Comment by ross-spencer** [comment]
I've merged that branch and released a suite [here](https://github.com/exponential-decay/pronom-archive-and-skeleton-test-suite/releases/tag/skeleton-test-suite-2020-01-20-sig-file-v96). I'm conscious the future of the suite will need to change with PRONOM/DROID changes. Do you have any thoughts on that, I'm split between:

* Stabilizing and productionizing the code already here.
* Rewriting. Maybe create a single generator binary. 

But I'm not sure.

**Comment by richardlehane** [comment]
I run both generator tools remotely thru a travis script with this [repo](https://github.com/richardlehane/builder). It's working well for me like that so from my perspective option (A) would be more than fine
